lib/chef-dk/skeletons/code_generator/recipes/build_cookbook.rb

```diff
@@ -169,7 +170,7 @@
   end
 
   execute("git-remove-delivery-config-branch") do
-    command("git branch -d add-delivery-configuration")
+    command("git branch -D add-delivery-configuration")
```

Comment:
Ok, I was just wondering because I chose not to use `-D` when I wrote this originally, but this explanation makes sense.